atomicapp/constants.py

```diff
@@ -1,5 +1,5 @@
-__ATOMICAPPVERSION__ = '0.0.1-alpha'
-__NULECULESPECVERSION__ = '0.0.1-alpha'
+__ATOMICAPPVERSION__ = '0.1.0'
+__NULECULESPECVERSION__ = '0.0.2'
```

Comment:
Hmm..the question is if the SCHEMA_URL needs to be there as it's not used anymore (was in create.py)